app/upgrades/v11/upgrades.go

```diff
@@ -9,36 +9,13 @@ import (
 	"github.com/osmosis-labs/osmosis/v10/app/upgrades"
 )
 
-// We set the app version to pre-upgrade because it will be incremented by one
-// after the upgrade is applied by the handler.
-const preUpgradeAppVersion = 10
-
 func CreateUpgradeHandler(
 	mm *module.Manager,
 	configurator module.Configurator,
 	bpm upgrades.BaseAppParamManager,
 	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
-	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-		// Although the app version was already set during the v9 upgrade, our v10 was a fork.
-		// As a result, the upgrade handler was not executed to increment the app version.
-		// This change helps to correctly set the app version for v11.
-		if err := keepers.UpgradeKeeper.SetAppVersion(ctx, preUpgradeAppVersion); err != nil {
-			return nil, err
-		}
-
-		// Set the max_age_num_blocks in the evidence params to reflect the 14 day
-		// unbonding period.
-		//
-		// Ref: https://github.com/osmosis-labs/osmosis/issues/1160
-		cp := bpm.GetConsensusParams(ctx)
-		if cp != nil && cp.Evidence != nil {
-			evParams := cp.Evidence
-			evParams.MaxAgeNumBlocks = 186_092
-
-			bpm.StoreConsensusParams(ctx, cp)
-		}
-
-		return mm.RunMigrations(ctx, configurator, fromVM)
+	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		return vm, nil
```

Comment:
Let's aim for this to be 1:1 compatible with `v11.x` please
https://github.com/osmosis-labs/osmosis/blob/a571be35d616c3cbbea87fdbc2bc115ca7ea60ba/app/upgrades/v11/upgrades.go#L17

Reply:
It appears between v11.x and main the actual CreateUpgradeHandler added a fourth input, so we cant be exactly 1:1 but I just made it as close as possible like you said, lmk if that meets intent!